hack/istio/README.adoc

```diff
@@ -10,10 +10,29 @@ This link:./istio-install.sh[install script] is used to deploy http://istio.io[I
 
 Simply run the script on a command line with no arguments to install the latest version of Istio on OpenShift.
 
+You'll need some dependencies for this to work:
+
+```bash
+sudo pip2 install jinja2 PyYAML
```

Comment:
I think this should be covered here: 

https://github.com/kiali/kiali/blob/master/hack/istio/istio-install.sh#L20

But maybe replace `pip` with `pip2` there, that way, if there is a new dependency, we won't have to touch this Readme

Reply:
and this one: https://github.com/kiali/kiali/blob/master/hack/istio/istio-install.sh#L26 (pip => pip2)

Reply:
I'm not sure, I had to install it globally to work, and the installed ansible is local. Not sure what should be done here.

Reply:
yeah, you shouldn't have to worry about telling the user to do this - that requirements.txt should do it.

Reply:
I guess that your python points to python3 and your pip points to pip2 maybe? (or some other combination), that thing is also installing the modules globally.

Reply:
@cfcosta I would remove this section (since users should not have to do this) but you can add another "troublehsooting" item to say, "the installer should be installing these dependencies, but if for some reason it fails to do so, you can run this command to install them youself....."